roles/network_plugin/calico/tasks/pre.yml

```diff
@@ -12,7 +12,9 @@
   - name: Set fact calico_datastore to etcd if needed
     set_fact:
       calico_datastore: etcd
-    when: "'etcd_endpoints' in calico_cni_config.plugins.0"
+    when: 
```

Comment:
Please check the failing job as https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/1352119588
The job says this line contains unnecessary space at the end of line:
```
roles/network_plugin/calico/tasks/pre.yml:15: [E201] Trailing whitespace
```